src/cms/pages-workspace.md

```diff
@@ -50,3 +50,14 @@ _Pages grid_
 |Created|The date the page was created.|
 |Modified|The date the page was last modified.|
 |Action|The actions that can be applied to an individual record include:<br/>**Edit** - Opens the page in edit mode.<br/>**Delete** - Deletes the page.<br/>**View** - Displays the page in preview mode.  |
+
+## Other columns
```

Comment:
What other columns are these? Are they enabled from the Columns control? If so we should explain that before presenting the table list.

Reply:
You're right! Done ✅